Makefile

```diff
@@ -26,11 +26,16 @@ all_generated_files := \
 .DEFAULT_GOAL := help
 
 ##@ Dependencies
+.PHONY: install-build-deps
```

Comment:
I'm not a `make` expert by any means so I could be wrong but should this be `install-all-deps`?

Reply:
`install-site-deps` looking further down?

Reply:
It was my mistake, corrected :)